code/modules/mob/living/simple_animal/abnormality/he/silent_girl.dm

```diff
@@ -86,7 +88,7 @@
 	if (!LAZYLEN(guilty_people)) // No Guilty on 0 counter? Find a random person and take them <3
 		for(var/mob/living/carbon/human/sucker in GLOB.mob_living_list)
 			if(!sucker.sanity_lost && sucker.stat != DEAD)
-				guilty_people += sucker
+				Guilt_Effect(sucker)
 				break
 	for(var/i in guilty_people) // Drive all Guilty people insane on Qliphoth 0
 		DriveInsane(i)
```

Comment:
This all could be reworked to pick someone randomly out of a list, instead of always picking the first one on the global list.
It would also be nice if DriveInsane was on timer, just so that there's a bit of time between seeing what's about to happen and it actually happening; This would both look better and provide target with an opportunity to yell "Going insane in control".